### PR TITLE
FIP 003: Lowering the staking requirements

### PR DESCRIPTION
**Pull request by LiorRabin**

### CODE REVIEW

**Comment by LiorRabin** [comment]
https://github.com/fuseio/FIPs/blob/master/FIPS/fip-3.md

File: test/contracts/ConsensusMock.sol [hunk 1]
```diff
@@ -39,6 +39,10 @@ contract ConsensusMock is Consensus {
     boolStorage[SHOULD_EMIT_INITIATE_CHANGE] = _status;
   }
 
+  function getMaxValidators() public pure returns(uint256) {
+    return 3;
```
Comment (bejavu) [inline test/contracts/ConsensusMock.sol]: did you test it with 100 as it should be?

File: contracts/ConsensusUtils.sol [hunk 2]
```diff
@@ -188,14 +206,6 @@ contract ConsensusUtils is EternalStorage, ValidatorSet {
...
   }
 
-  function _setSnapshotStakeAmountForAddress(uint256 _snapshotId, address _address) internal {
-    uintStorage[keccak256(abi.encodePacked("snapshot", _snapshotId, "address", _address, "stakeAmount"))] = stakeAmount(_address);
-  }
-
-  function _getSnapshotStakeAmountForAddress(uint256 _snapshotId, address _address) internal view returns(uint256) {
-    return uintStorage[keccak256(abi.encodePacked("snapshot", _snapshotId, "address", _address, "stakeAmount"))];
-  }
-
```
Comment (bejavu) [inline contracts/ConsensusUtils.sol]: why did you removed this, was those functions ever been used?
are stake should be exactly 100K?

Comment (LiorRabin) [reply contracts/ConsensusUtils.sol]: Never used, probably leftovers from a long time ago...
Stake should be exactly 100k, can't stake more

File: test/contracts/ConsensusMock.sol (hunk 1 above)
Comment (LiorRabin) [reply test/contracts/ConsensusMock.sol]: Just manually the function itself, but I will add a test

**Pull request merged**